_posts/2021-09-28-str-format.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "String Formatting"
+date: 2021-09-28
+author: Arjun
+category: Tutorial
+---
+
+<h2>Here is the list of complete set of symbols which can be used along with "%"</h2>
```

Comment:
You can use "##" instead of <h2>. Does the same. But, easier to write using Markdown syntax